xla/service/gpu/transforms/command_buffer_scheduling.cc

```diff
@@ -244,7 +243,8 @@ static bool IsCommand(const HloInstruction* hlo,
       return config.enabled_commands.contains(DebugOptions::CUDNN);
     }
     const auto& custom_config = backend_config.custom_fusion_config();
-    if (custom_config.name() == "address_computation") {
+    if ((custom_config.name() == "address_computation") ||
```

Comment:
is it still adddress computation? we were replacing it with slice/dynamic_slice, but might forgot a few places

Reply:
Seems that the config name still has `address_computation` as name, we can have another PR to change the name?